src/map/mob.cpp

```diff
@@ -1871,12 +1919,10 @@ static bool mob_ai_sub_hard(struct mob_data *md, t_tick tick)
 	{
 		int32 prev_id = md->target_id;
 		map_foreachinallrange (mob_ai_sub_hard_activesearch, &md->bl, view_range, DEFAULT_ENEMY_TYPE(md), md, &tbl, mode);
-		// If a monster finds a target through search that is already in attack range it immediately switches to berserk mode
+		// If a monster finds a target through search that is already in attack range it switches to berserk mode
```

Comment:
```suggestion
		// If a monster finds a target through search, that is already in attack range, it switches to berserk mode
```

Reply:
I don't think that's correct English punctuation, especially not in front of a "that": 
https://editorsmanual.com/articles/commas-with-that-which/

You can optionally set a comma in front of the second part of an if-sentence in English, but that's also optional:
https://grammarhow.com/comma-before-if/